test/SparqlAntlrParserTest.cpp

```diff
@@ -1000,8 +1000,7 @@ TEST(SparqlParser, GroupGraphPattern) {
       m::GraphPattern(m::Service(Iri{"<ep>"}, {Var{"?s"}, Var{"?o"}},
                                  "{ { SELECT ?s ?o WHERE { ?s ?p ?o } } }")));
 
-  // SERVICE with SILENT or a variable endpoint is not yet supported.
-  expectGroupGraphPatternFails("{ SERVICE SILENT <ep> { ?s ?p ?o } }");
+  // SERVICE with a variable endpoint is not yet supported.
```

Comment:
You should extend the `m::Service` matcher, s.t. it also takes and checks the information about `SILENT` 
(you can default it to `not Silent`, so you only have to change the test for silent.

Reply:
(The silent case is uncovered, but codecov can't report because of your `static_cast<bool>(SILENT)` which hides the branch, so we have to remember this manually:)